setup.py

```diff
@@ -9,7 +9,7 @@
     python_requires=">=3.8",
     install_requires=[
         "watertap @ https://github.com/watertap-org/watertap/archive/refs/tags/pr967.zip",
-        "pyomo <= 6.5",
+        # "pyomo <= 6.5",
```

Comment:
For the final merge of this PR, we should specify the actual version of Pyomo that this is running against, i.e., 
`pyomo <= 6.6.1` (or whatever version it is we're looking at).

Reply:
Pyomo version is specified now.

Aslo I noticed there's other packages that have a different version from the previous environment, for example `pytest_cov == 4.1.0` vs. previous 4.0.0. Not sure if those packages will change the solvability of the model, but `pyomo` should be the major one.

Reply:
I think that corresponds to pytest's code coverage and wouldn't expect it to change solvability of the model. @lbianchi-lbl would know best about any caveats related to that in general though.